### PR TITLE
Export all

### PR DESCRIPTION
**Pull request by shakalee14**
@GeneralMeow 
@sdweber422 
Fix #108

### CODE REVIEW

**Comment by shakalee14** [comment]
@GeneralMeow @sdweber422 
take a look

File: views/layouts/authenticated-layout.pug [hunk 1]
```diff
@@ -41,16 +42,22 @@ html
                   .dropdown__item--label#weekly-summary Weekly Summary
               li.dropdown__item
                 a.dropdown__item--link(href='/accounts/logout')
+              li.dropdown__item
+                a.dropdown__item--export-link Export All
+              li.dropdown__item
+                a.dropdown__item--link(href='/accounts/logout')
```
Comment (GeneralMeow) [inline views/layouts/authenticated-layout.pug]: should there be two dropdown menus for link(href='/accounts/logout') ?

File: views/layouts/authenticated-layout.pug [hunk 2]
```diff
@@ -41,16 +42,22 @@ html
...
+
     include ../accounts/weekly-report.pug
 
     script(src='https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js')
     script(src='/javascripts/fetch-params.js')
     script(src='/javascripts/items.js')
     script(src='/javascripts/starred.js')
     script(src='/javascripts/weekly-review.js')
+    script(src='/javascripts/client.js')
+    script(src='/javascripts/client.js')
```
Comment (GeneralMeow) [inline views/layouts/authenticated-layout.pug]: just wondering why it's added here twice 

+    script(src='/javascripts/client.js')
 +    script(src='/javascripts/client.js')

**Comment by shakalee14** [comment]
@jrob8577

**Comment by tjfwalker** [comment]
@jrob8577 assign me to this, please.

**Pull request merged**